eth/rollup.go

```diff
@@ -814,6 +814,14 @@ func (c *RollupClient) RollupEventsByBlock(blockNum int64,
 			L1Tx.ToForgeL1TxsNum = &toForgeL1TxsNum
 			L1Tx.Position = int(new(big.Int).SetBytes(vLog.Topics[2][:]).Int64())
 			L1Tx.UserOrigin = true
+			L1Tx.EthTxHash = vLog.TxHash
+			//Get l1Fee in eth wei spent in the l1 tx
+			tx, _, err := c.client.client.TransactionByHash(context.Background(), vLog.TxHash)
+			if err != nil {
+				return nil, tracerr.Wrap(fmt.Errorf("TransactionByHash: %w", err))
```

Comment:
Maybe it will be better to provide a more descriptive error message?
```suggestion
				return nil, tracerr.Wrap(fmt.Errorf("failed to get TransactionByHash, hash: %s, err: %w", vLog.TxHash.String(), err))
```